### ISSUE TITLE
New release on Pypi

### ISSUE THREAD

**Issue by Tomo59** [Supervisor/supervisor#1635]
Hello, could you do a new release on pypi ? It's been more than a year now and there have been several bugfixes included in main branch since then.

Thanks.

Thomas.

**Comment by sbraz** [comment]
Hi, I second this request. The latest release has a lot of warnings that were fixed in git like `DeprecationWarning: unittest.findTestCases() is deprecated and will be removed in Python 3.13.`.

**Comment by mnaberez** [comment]
> The latest release has a lot of warnings that were fixed in git like DeprecationWarning: unittest.findTestCases() is deprecated and will be removed in Python 3.13.

Yeah, looks like that was fixed 49b74cafb6e72e0e620e321711c1b81a0823be12, just a couple days after 4.2.5 was released to PyPI.

**Comment by JKrehling** [comment]
Theres been some CPU fixes too, https://github.com/Supervisor/supervisor/commit/2a93d6b21edca4e8b00987c70a350ca626aebec8

What would be needed to get a 4.3.0 or a 4.2.6 release on pypi?

**Comment by whatnick** [comment]
Perhaps a concrete action here would be to create a PR for pypi release from GitHub actions to reduce the manual load on the maintainers. That way a tag can create a release (at least a dev one on test.pypi).

**Comment by dkebler** [comment]
It has been almost two years of commits since 4.2.5.  Not sure if I need them but I only like to use projects that are relatively up to date and active.  This project is active but the release is out of date.  I assume there is something like a stable release in these commits somewhere.    Not really knowing which commit  I decided to just clone the latest commit and run that to see assumming all the commits are vetted/tested.

On alpine (in a container)  I did this.

first install the supervisor package

then

```
mv /usr/lib/python3.12/site-packages/supervisor  /usr/lib/python3.12/site-packages/supervisor-4.2.5
git clone git clone https://github.com/Supervisor/supervisor.git /tmp/supervisor
mv /tmp/supervisor/supervisor /usr/lib/python3.12/site-packages/supervisor
supervisord --version
```

If the maintainers ask for some help maybe we can assist with clearing/closing all the old issues and assuming this repo is not just a mirror making a CI for a stable/dev releases.   I figure the alpine package generator is only waiting on a new stable tag/release and then we would see the new version in the package so that's why it's so stale at this point.

**Comment by sbraz** [comment]
Hello, any news on this?

**Comment by xiaobao0505** [comment]
Hello,  Any progress?

**Comment by cfsnate** [comment]
+1

**Comment by mroland91** [comment]
+1

**Comment by number492** [comment]
+1

**Comment by number492** [comment]
@mnaberez what needs to happen for a new release to hit Pypi? Can I help in any way?

**Comment by chrisgoddard** [comment]
hey there @mnaberez  - any updates on this? Anything we can do to help?

**Comment by alexandrul** [comment]
You can use a specific commit in `requirements.txt`:

`supervisor @ git+https://github.com/Supervisor/supervisor.git@4bf1e57cbf292ce988dc128e0d2c8917f18da9be
`
